.travis.yml

```diff
@@ -24,6 +26,22 @@ matrix:
     - env: PYTHON_VM=ipy OPTIONAL_DEPS=no
     - python: "pypy"
 
+addons:
+  apt:
+    sources:
+    - ubuntugis/ubuntugis-unstable
```

Comment:
Probably we don't need this. The whitelist https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise may suffice.